### PR TITLE
minor edits to fee_bumping.md

### PR DESCRIPTION
**Pull request by adamjonas**
A few minor edits -- I see the add_rbf branch but these are edits off of master.

I also noticed that the usage of feerate is spelled differently through the doc (in ascending order of usage: fee rate, fee-rate, and feerate).

### CODE REVIEW

File: 1.fee_bumping/fee_bumping.md [hunk 1]
```diff
@@ -321,11 +321,11 @@ their CPFP system:
   do the same process when creating a 3rd or 4th generation transaction to pay
   for its ancestors.
 - the Bitcoin Core mining algorithm will only consider packages of up to 25
-  transactions or 101vkB. HBE therefore needs to make sure they're not creating
+  transactions or 101vkB. HBE, therefore, needs to make sure they're not creating
```
Comment (jnewbery) [inline 1.fee_bumping/fee_bumping.md]: Again, I don't think these are needed. I'm with the Chicago Manual of Style 5.69 on this one: https://english.stackexchange.com/a/40241/83566

File: 1.fee_bumping/fee_bumping.md [hunk 2]
```diff
@@ -382,8 +382,8 @@ transactions.
 ## Consensus, policy and incentive compatibility
 
 Both solutions discussed in this article are related to network node and miner
-behavior before a transaction is included in a block. That behavior is
-therefore a question of policy rather than consensus. Both solutions are also
+behavior before a transaction is included in a block. That behavior is,
```
Comment (jnewbery) [inline 1.fee_bumping/fee_bumping.md]: Same as above!

File: 1.fee_bumping/fee_bumping.md [hunk 3]
```diff
@@ -179,7 +179,7 @@ Even wallets and services that do not themselves support creating opt-in RBF or
 replacement transactions should present a clear and accurate experience to
 their users when dealing with RBF transactions:
 
-- wallets that receive transactions that have opt-in RBF signalled may
+- wallets that receive transactions that have opt-in RBF signaled may
```
[the remark this answers is not in the document]
Comment (adamjonas) [reply 1.fee_bumping/fee_bumping.md]: Understood. Just commenting on consistency since behaviour and other Britishisms have been Americanized.

**Comment by jnewbery** [comment]
ACK abe64db8ef44df109491c6d54b799f9d7c254117. Thanks!

**Pull request merged**